src/config/index.ts

```diff
@@ -152,6 +152,10 @@ const config = (): Configuration => {
 		},
 		MEEMOO_ADMIN_ORGANIZATION_IDS: getEnvValue('MEEMOO_ADMIN_ORGANIZATION_IDS', true) || '',
 		REROUTE_EMAILS_TO: getEnvValue('REROUTE_EMAILS_TO', false),
+		MEEMOO_MAINTAINER_MISSING_EMAIL_FALLBACK: getEnvValue(
+			'MEEMOO_MAINTAINER_MISSING_EMAIL_FALLBACK',
+			false
```

Comment:
required: true since otherwise the material requests can sometimes just fail
```suggestion
			true
```